sdk/js-query/src/query/solana.test.ts

```diff
@@ -27,6 +27,7 @@ const ENV = "DEVNET";
 const SERVER_URL = CI ? "http://query-server:" : "http://localhost:";
 const CCQ_SERVER_URL = SERVER_URL + "6069/v1";
 const QUERY_URL = CCQ_SERVER_URL + "/query";
+const SOLANA_NODE_URL = CI ? "http://solana-devnet" : "http://localhost:8899";
```

Comment:
I think this needs to be `const SOLANA_NODE_URL = CI ? "http://solana-devnet:8899" : "http://localhost:8899";`